In-house patch to remove unnecessary dependencies from Glance's
requirements files. The specific reasons are as follows:

boto		Not applicable

cinderclient	Not applicable

greenlet	Not applicable

kombu		Not applicable

oslo.vmware	Not applicable

pycrypto	Not applicable to Solaris (M2Crypto used instead)

swiftclient	Not applicable

--- glance-2014.2.2/glance.egg-info/requires.txt.~1~	2015-02-05 07:21:46.000000000 -0800
+++ glance-2014.2.2/glance.egg-info/requires.txt	2015-02-23 14:39:12.013101917 -0800
@@ -1,5 +1,4 @@
 pbr>=0.6,!=0.7,<1.0
-greenlet>=0.3.2
 SQLAlchemy>=0.8.4,<=0.9.99,!=0.9.0,!=0.9.1,!=0.9.2,!=0.9.3,!=0.9.4,!=0.9.5,!=0.9.6
 anyjson>=0.3.3
 eventlet>=0.15.1,<0.16.0
@@ -7,11 +6,8 @@ PasteDeploy>=1.5.0
 Routes>=1.12.3,!=2.0
 WebOb>=1.2.3
 argparse
-boto>=2.32.1,<2.35.0
 sqlalchemy-migrate==0.9.1
 httplib2>=0.7.5
-kombu>=2.5.0
-pycrypto>=2.6
 iso8601>=0.1.9
 ordereddict
 oslo.config>=1.4.0  # Apache-2.0
@@ -20,11 +16,8 @@ netaddr>=0.7.12
 keystonemiddleware>=1.0.0
 WSME>=0.6
 posix_ipc
-python-swiftclient>=2.2.0
-oslo.vmware>=0.6.0,<0.9.0               # Apache-2.0
 Paste
 jsonschema>=2.0.0,<3.0.0
-python-cinderclient>=1.1.0
 python-keystoneclient>=0.10.0
 pyOpenSSL>=0.11
 six>=1.7.0
--- glance-2014.2.2/requirements.txt.~1~	2015-02-05 07:19:44.000000000 -0800
+++ glance-2014.2.2/requirements.txt	2015-02-23 14:39:03.037319077 -0800
@@ -3,11 +3,6 @@
 # process, which may cause wedges in the gate later.
 
 pbr>=0.6,!=0.7,<1.0
-#
-# The greenlet package must be compiled with gcc and needs
-# the Python.h headers. Make sure you install the python-dev
-# package to get the right headers...
-greenlet>=0.3.2
 
 # < 0.8.0/0.8 does not work, see https://bugs.launchpad.net/bugs/1153983
 SQLAlchemy>=0.8.4,<=0.9.99,!=0.9.0,!=0.9.1,!=0.9.2,!=0.9.3,!=0.9.4,!=0.9.5,!=0.9.6
@@ -17,11 +12,8 @@ PasteDeploy>=1.5.0
 Routes>=1.12.3,!=2.0
 WebOb>=1.2.3
 argparse
-boto>=2.32.1,<2.35.0
 sqlalchemy-migrate==0.9.1
 httplib2>=0.7.5
-kombu>=2.5.0
-pycrypto>=2.6
 iso8601>=0.1.9
 ordereddict
 oslo.config>=1.4.0  # Apache-2.0
@@ -32,17 +24,10 @@ WSME>=0.6
 # For openstack/common/lockutils
 posix_ipc
 
-# For Swift storage backend.
-python-swiftclient>=2.2.0
-
-# For VMware storage backed.
-oslo.vmware>=0.6.0,<0.9.0               # Apache-2.0
-
 # For paste.util.template used in keystone.common.template
 Paste
 
 jsonschema>=2.0.0,<3.0.0
-python-cinderclient>=1.1.0
 python-keystoneclient>=0.10.0
 pyOpenSSL>=0.11
 
